app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1142,6 +1142,49 @@ class PatientRepositoryAndroidTest {
     }
   }
 
+  @Test
+  fun verify_deleted_Appointments_are_not_included_when_fetching_recent_patients() {
```

Comment:
Rename method to `verify_deleted_appointments_are_not_included_when_fetching_recent_patients()`

Reply:
done